### DISCUSSION TITLE
Руководство ПЛИ, тело документа, п.1.6 Секция "Оказанные услуги" SERVICES

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#285] [Лабораторная РГ]
У3-5
Секция предназначена для указания сведений о медицинских услугах, оказанных в рамках проведения лабораторных исследований, описанных в данном документе.
В данную секцию включается следующая информация:
• Оказанная услуга;
• Дата и время оказания услуги.

**Comment by eugenekogan-spb** [comment]
Вопрос: чем отличается информация в данной секции от информации в разделе "Медицинские услуги" секции "Результаты".

**Comment by eugenekogan-spb** [comment]
<img width="753" alt="image" src="https://github.com/user-attachments/assets/3feb8309-eed4-43c5-913f-2f413b11f897">

**Reply by eugenekogan-spb** [reply]
Согласно решениям, описанным [в разделе о выполненных услугах секции Результаты](https://github.com/fhir-ru/core/discussions/284#discussioncomment-11091583), сведения об оказанных услугах передаются в расширении ресурса DiagnosticReport. Поскольку данная секция предполагает передачу даты оказания услуги, следует предусмотреть для этого в расширении соответствующий атрибут. 

```
* extension[+]
  * url = "procedureCode"
  * valueCodeableConcept
    * text = "Общий (клинический) анализ крови"
    * coding
      * system = "1.2.643.5.1.13.13.11.1070"
      * code = #B03.016.002
  * extension[+]
    * url = "Date"
    * valueDateTime = "2022-11-15T08:50:00+04:00"
```

**Reply by maximalexeev** [reply]
А почему не DiagnosticReport.code? дата в ДР есть.

**Reply by eugenekogan-spb** [reply]
потому что иначе пришлось бы делить DR по услугам, что клинически некорректно. Решили, что деление протокола на DR происходит по профилям исследований, то есть по группам тестов, имеющих однотипное происхождение.
